styles.css

```diff
@@ -198,38 +198,30 @@ h3 {
 .necklace {
   height: 13%;
   left: 31%;
-  position: absolute;
```

Comment:
this change and the others in the css were necessary because we forgot to push these changes to the master branch when we did it at a previous step.